### PR TITLE
IN PROCESS - Partial mobile

### PR DESCRIPTION
**Pull request by gcwokeup**
**Desktop User Header**
image
.
.
.
.
**Mobile(Medium/tablet size and below) User Header**
image

### CODE REVIEW

File: app/assets/stylesheets/layouts/user_header.scss [hunk 1]
```diff
@@ -18,11 +22,35 @@
...
+      @include media-breakpoint-down(md) {
+        width: 100%;
+      }
+    }
   }
 
   &__location {
     color: $text-toned-down;
     margin-bottom: 4px;
+
```
Comment (markdamobi) [inline app/assets/stylesheets/layouts/user_header.scss]: hmmm...

**Comment by markdamobi** [comment]
looks good. I'm not going to mark as approved since the title still says "in-process". I'm assuming more work is still being done.

File: app/assets/stylesheets/layouts/user_header.scss (hunk 1 above)
Comment (gcwokeup) [reply app/assets/stylesheets/layouts/user_header.scss]: my bad forgot to take out IN-PROCESS

**Comment by markdamobi** [comment]
@gcwokeup I just noticed(compared to [this](https://app.zeplin.io/project/5c5390a362086d356556a647/screen/5c7ec43b3ff28062623e0a7a)) that when it's in mobile view, the location and social icons disappear. Is that something you did on this branch, or is it part of another branch?

**Comment by markdamobi** [comment]
> @gcwokeup I just noticed(compared to [this](https://app.zeplin.io/project/5c5390a362086d356556a647/screen/5c7ec43b3ff28062623e0a7a)) that when it's in mobile view, the location and social icons disappear. Is that something you did on this branch, or is it part of another branch?

looks like the "Edit Profile" button also disappears in the mobile view. Not sure why.

**Comment by gcwokeup** [comment]
@markdamobi good catch. we were following the user-home mobile version. Didn't look at the othe r pages. I'll put this back to IN PROCESS and fix it

**Comment by sanghun89** [comment]
looks good so far

**Pull request merged**